src/main/webapp/admin_dashboard.html

```diff
@@ -1,122 +1,126 @@
+		<link rel="stylesheet" href="admin_dashboard_style.css">
+		<script src='https://kit.fontawesome.com/a076d05399.js'></script>
+		<script src="https://www.gstatic.com/charts/loader.js"></script>
+		<script type="text/javascript" src="config.js"></script>
+		<script src="admin_dashboard_script.js"></script>
+	</head>
+	<body>
+		<div class="wrapper">
+			<main>
+				<h1>Admin Prototype</h1>
+				<div class="flex-grid">
+					<div class="card-container">
+						<h2>Administrator Tasks</h2>
+						<div id="card-div">
+							<ul id="admin-tasks">
+								<li class="admin-task">
+									<h3>New Homepage Design</h3>
+									<h4>Mafe Ponce</h4>
+									<div class="admin-task-infos">
+										<p class="date"><i class="far fa-calendar"></i> 06/26/2020</p>
+										<p class="time"><i class="far fa-clock"></i> 3:30pm</p>
+									</div>
+								</li>
+								<li class="admin-task">
+									<h3>User Tasks Design</h3>
+									<h4>Leonard Zhang</h4>
+									<div class="admin-task-infos">
+										<p class="date"><i class="far fa-calendar"></i> 06/26/2020</p>
+										<p class="time"><i class="far fa-clock"></i> 3:30pm</p>
+									</div>
+								</li>
+								<li class="admin-task">
+									<h3>Admin Console Design</h3>
+									<h4>Denzil Ekow Bilson</h4>
+									<div class="admin-task-infos">
+										<p class="date"><i class="far fa-calendar"></i> 06/26/2020</p>
+										<p class="time"><i class="far fa-clock"></i> 3:30pm</p>
+									</div>
+								</li>
+								<li class="admin-task">
+									<h3> User Authentication</h3>
+									<h4>Mafe Ponce</h4>
+									<div class="admin-task-infos">
+										<p class="date"><i class="far fa-calendar"></i> 06/30/2020</p>
+										<p class="time"><i class="far fa-clock"></i> 4:00pm</p>
+									</div>
+								</li>
+								<li class="admin-task">
+									<h3>404 Page Design</h3>
+									<h4>Denzil Bilson</h4>
+									<div class="admin-task-infos">
+										<p class="date"><i class="far fa-calendar"></i> 06/30/2020</p>
+										<p class="time"><i class="far fa-clock"></i> 4:00pm</p>
+									</div>
+								</li>
+								<li class="admin-task">
+									<h3>Upload Tasks</h3>
+									<h4>Leonard Zhang</h4>
+									<div class="admin-task-infos">
+										<p class="date"><i class="far fa-calendar"></i> 06/30/2020</p>
+										<p class="time"><i class="far fa-clock"></i> 4:00pm</p>
+									</div>
+								</li>
+							</ul>
+						</div>
+					</div>
+					<div id="chart-container">
+						<h2>Usage Statistics</h2>
+						<div id="chart-div"></div>
+					</div>
+				</div>
+				<div class="flex-grid">
+					<div id="map-container">
+						<h2>Map</h2>
+						<div id="map-div"></div>
+					</div>
+					<div class="card-container">
+						<h2> Last 10 User Tasks</h2>
+						<div id="card-div">
+							<ul id="user-tasks">
+							</ul>
+						</div>
+					</div>
+				</div>
+			</main>
+		</div>
+		<div id="popup-overlay">
```

Comment:
Is it possible to change the style of the edit task popup window? The color scheme can stay the same but I think it would be better if the edit task modal has the same size as other pages' pop up window

Reply:
The styles across pages will be dealt with after the MVP